### PR TITLE
Initialize CAAM drivers with job ring base address

### PR DESCRIPTION
**Pull request by clementfaure**
Hello,

A patch set to initialize CAAM hash, ecc, rsa and math drivers with the given JR base address instead of taking JR0 as base address systematically.
It prepares the CAAM support for imx8q/qxp

Thanks!
Clement

### CODE REVIEW

File: core/drivers/crypto/caam/include/caam_hash.h [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 #ifndef __CAAM_HASH_H__
 #define __CAAM_HASH_H__
 
+#include <caam_jr.h>
+
 #ifdef CFG_NXP_CAAM_HASH_DRV
 /*
  * Initialize the Hash module
  *
  * @ctrl_addr   Controller base address
```
Comment (jforissier) [inline core/drivers/crypto/caam/include/caam_hash.h:16]: You forgot to update the comment (same thing below)

**Review by jforissier — COMMENTED** [review]
With the comments updated:
`Acked-by: Jerome Forissier <jerome@forissier.org>`

File: core/drivers/crypto/caam/include/caam_hash.h (hunk 1 above)
Comment (clementfaure) [reply core/drivers/crypto/caam/include/caam_hash.h]: Fixed!

**Comment by clementfaure** [comment]
Force-push: comment addressed and tag applied

**Comment by clementfaure** [comment]
Force-push: remove an unnecessary copyright update in `caam_cipher.h`

**Pull request merged**